Libraries/Lists/VirtualizedList.js

```diff
@@ -1592,9 +1592,25 @@ export default class VirtualizedList extends StateSafePureComponent<
     // starving the renderer from actually laying out the objects and computing _averageCellLength.
     // If this is triggered in an `componentDidUpdate` followed by a hiPri cellToRenderUpdate
     // We shouldn't do another hipri cellToRenderUpdate
+
+    // We should trigger only high pirority updates on area with cells already measured or if measure new cells
+    // is keeping it up with scroll toward unmeasured area. Otherwise scrolling too fast toward unmeasured area
+    // while new items are being measured may skip cells to be measured and cause scroll issues on scroll back to skipped area.
+    // 'visibleLength / 2' represents a balanced ‘bar check’ to be sure that all items
+    // are gonna be measured and mounted as fast as possible when user scrolls quickly towards unmeasured area.
```

Comment:
nit: spelling/typos and grammar

E.g. "We should trigger only high priority updates" reads as "we should never trigger low priority updates", where I think this was meant to be "We should trigger high priority updates only if".